gnat2goto/driver/tree_walk.adb

```diff
@@ -4244,81 +4269,247 @@ package body Tree_Walk is
       end;
    end Maybe_Make_Typecast;
 
+   --------------------------------
+   --  Warn_Unhandled_Construct  --
+   --------------------------------
+
+   procedure Warn_Unhandled_Construct (C : Construct; Mess : String) is
+      S : constant String :=
+        (case C is
+            when Declaration => " declarations ",
+            when Statement   => " statements ") & "unhandled";
+   begin
+      Put_Line (Standard_Error, "Warning: " & Mess & S);
+   end Warn_Unhandled_Construct;
+
+   --------------------------
+   -- Process_Declaration --
+   --------------------------
+
+   procedure Process_Declaration (N : Node_Id; Block : Irep) is
```

Comment:
Shouldn't this be called `Do_Declaration`?

Reply:
Hello I'm back - only just. I was hoping to resolve this merge today but it seems that you are doing it for me! Thanks. 
I am not sure of the naming convention you are using I started calling the procedure Do_Declaration but then changed it Process_Declaration because I basically took the code from Process_Statement.
Do you have a convention for naming.  If so what is it and please call this procedure accordingly.

Reply:
I think that `Do_*` return `Irep`.